apps/swirl-docs/src/lib/docs/src/multipleDocs.ts

```diff
@@ -1,108 +1,122 @@
+import fs from "fs";
+import { GetStaticPathsResult } from "next";
 import {
   API_SPEC_PATH,
   generateDocumentationPath,
   generatePagesPath,
 } from "@swirl/lib/navigation";
-import fs from "fs";
-import { GetStaticPathsResult } from "next";
+import { apiSpecsNavItems } from "@swirl/lib/navigation/src/data/apiSpecs.data";
 import {
   DocCategory,
   Document,
   DocumentationCategory,
   DOCUMENTATION_SRC,
   StaticPathMapType,
 } from "./docs.model";
-import { apiDocsNavItems } from "@swirl/lib/navigation/src/data/apiDocs.data";
 
 export const StaticPathMap: StaticPathMapType = {
   components: "componentDoc",
   icons: "iconDoc",
   tokens: "tokenDoc",
-  apiDocs: "apiDoc",
+  api: "apiDoc",
 } as const;
 
+/***********************************************
+ * Helper Functions
```

Comment:
**suggestion:** Structuring comments are often a sign that the file should be split. ;)

Reply:
yeah normally i agree, but i think this file is not thaaaat big and having two not that big files seems off :) i would leave it as is for now :)